src/globals/test-extensions.php

```diff
@@ -40,7 +38,7 @@
 
 // If you want to test your added extensions and libs, add below (comma separated, example `bcmath,openssl`).
 $extensions = match (PHP_OS_FAMILY) {
-    'Linux', 'Darwin' => 'gettext',
+    'Linux', 'Darwin' => 'imap,openssl,zlib,memcache',
```

Comment:
I would take imap out here and turn zts back on. imap will at some point be removed from php entirely, zts is the future with more and more SAPIs embracing a worker model